script/import_db.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
I have moved `create_test_db.sh`. Is it preferable to move `script/create_test_db.sh` to `docker/scripts/`?